air/src/trace/mod.rs

```diff
@@ -14,7 +14,12 @@ pub mod stack;
 // ================================================================================================
 
 /// The minimum length of the execution trace. This is the minimum required to support range checks.
-pub const MIN_TRACE_LEN: usize = 64;
+/// Minimum trace length required by FRI parameters.
+///
+/// FRI requires: log2(MIN_TRACE_LEN) > log_final_poly_len + log_blowup
+/// With log_final_poly_len=7 and log_blowup=3, we need: log2(MIN_TRACE_LEN) > 10
+/// Therefore: MIN_TRACE_LEN >= 2048
+pub const MIN_TRACE_LEN: usize = 2048;
```

Comment:
Having such trace length is not a problem - except maybe for tests.

But generally, for small traces, we can just send the remainder polynomial, right?  I'm assuming this is not currently supported by Plonky3 FRI implementation - but maybe this is something we'll be able to do with Lifted FRI? Should we create an issue for this?

Reply:
I _think_ Lifted FRI supports this out of the box, ie we don't send any FRI commitments if the trace is shorter than the desired remainder polynomial.